baseline.yaml

```
@@ -649,6 +649,33 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+      - id: OSPS-DO-18   
```

Watch the indent here

```suggestion
  - id: OSPS-DO-18   
```